### PR TITLE
add Bring! integration

### PR DESCRIPTION
**Pull request by miaucl**
## Proposed change

New integration for to-do list [Bring!](https://www.getbring.com/en/home).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108027
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5149
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
@miaucl Small note: It seems like a PR against our brands repository is missing?

../Frenck

**Comment by tr4nt0r** [comment]
> @miaucl Small note: It seems like a PR against our brands repository is missing?
> 
> ../Frenck

I created the PR in the brands repo for @miaucl 

https://github.com/home-assistant/brands/pull/5149

File: source/_integrations/bring.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ha_codeowners:
+  - '@miaucl'
+  - '@tr4nt0r'
+ha_domain: bring
+ha_integration_type: integration
+ha_platforms:
+  - todo
+---
+
+The `bring` **to-do list** allows you to interact with your [Bring!](https://www.getbring.com/) shopping lists from within Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/bring.markdown]:
```suggestion
The **Bring** integration allows you to interact with your [Bring!](https://www.getbring.com/) shopping lists within Home Assistant.
```

File: source/_integrations/bring.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+The authentication uses directly your `email` / `password` and if you want to automatically receive notifications via the Bring! App when Home Assistant adds or removes an item from the list, you should use a dedicated account (such as `email: your.name+ha@gmail.com`) to connect Home Assistant with [Bring!](https://www.getbring.com/).
```
Comment (c0ffeeca7) [inline source/_integrations/bring.markdown]:
```suggestion
For authentication, the integration requires the `email` and `password` you used for your Bring! account. If you want to automatically receive notifications via the Bring! app when Home Assistant adds or removes an item from the list, you should use a dedicated account (such as `email: your.name+ha@gmail.com`) to connect Home Assistant with [Bring!](https://www.getbring.com/).
```

**Review by c0ffeeca7 — COMMENTED** [review]
Small tweaks

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @miaucl :+1:

**Pull request merged**